src/buffer/out/OutputCellIterator.hpp

```diff
@@ -100,6 +100,8 @@ class OutputCellIterator final
 
     bool _TryMoveTrailing() noexcept;
 
+    _declspec(noinline) void _MoveSlowPath() noexcept;

```

Comment:
Wait, `_declspec` works the same as `__declspec`?... I need this to be not-inlined to reduce the instruction count.